Microsoft.Toolkit.Uwp.UI/Cache/CacheBase.cs

```diff
@@ -79,17 +89,58 @@ public int MaxMemoryCacheCount
         }
 
         /// <summary>
+        /// Gets instance of <see cref="HttpClient"/>
+        /// </summary>
+        protected HttpClient HttpClient
+        {
+            get
+            {
+                if (_httpClient != null)
+                {
+                    return _httpClient;
+                }
+
+                var messageHandler = new HttpClientHandler() { MaxConnectionsPerServer = 10 };
```

Comment:
wouldn't we want to allow a developer to specify their own MaxConnectionsPerServer value if 10 didn't work for their scenario?